modules/playing-cards/widgets/cards-widget.php

```diff
@@ -0,0 +1,312 @@
+<?php
+namespace Elementor\Modules\PlayingCards\Widgets;
+
+use Elementor\Controls_Manager;
+use Elementor\Repeater;
+use Elementor\Widget_Base;
+use Elementor\Group_Control_Image_Size;
+
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class CardsWidget extends Widget_Base {
+
+	protected $card_suites_map = [
+		'clubs' => [
+			'name' => 'Clubs',
+			'symbol' => '&#9827;',
+			'color' => 'black'
+		],
+		'diamonds' => [
+			'name' => 'Diamonds',
+			'symbol' => '&#9830;',
+			'color' => 'red'
+		],
+		'hearts' => [
+			'name' => 'Hearts',
+			'symbol'=> '&#9829;',
+			'color' => 'red'
+		],
+		'spades' => [
+			'name' => 'Spades',
+			'symbol' => '&#9824;',
+			'color' => 'black'
+		]
+	];
+
+	/**
+	 * Get widget name.
+	 *
+	 * Retrieve playing cards widget name.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget name.
+	 */
+	public function get_name() {
+		return 'playing-cards';
+	}
+
+	/**
+	 * Get widget title.
+	 *
+	 * Retrieve playing cards widget title.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget title.
+	 */
+	public function get_title() {
+		return esc_html__( 'Playing cards', 'elementor' );
+	}
+
+	/**
+	 * Get widget icon.
+	 *
+	 * Retrieve playing cards widget icon.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget icon.
+	 */
+	public function get_icon() {
+		return 'eicon-number-field';
+	}
+
+	/**
+	 * Get widget keywords.
+	 *
+	 * Retrieve the list of keywords the widget belongs to.
+	 *
+	 * @since 2.1.0
+	 * @access public
+	 *
+	 * @return array Widget keywords.
+	 */
+	public function get_keywords() {
+		return [ 'playing cards', 'card', 'cards', 'play' ];
+	}
```

Comment:
What is it used for?

Reply:
no idea tbh, i thought initially that it's for the search